modules/markup/markdown/markdown_test.go

```diff
@@ -222,7 +229,7 @@ See commit 65f1bf27bc
 Ideas and codes
 
 - Bezier widget (by @r-lyeh) ` + AppURL + `ocornut/imgui/issues/786
-- Bezier widget (by @r-lyeh) ` + AppURL + `gogits/gogs/issues/786
+- Bezier widget (by @r-lyeh) ` + AppURL + `user13/repo11/issues/786
```

Comment:
Why `AppURL` is used but `testRepoOwnerName` not? 😕

Reply:
0815c2910d6e4c8149c44966b12901943ce918a8